Improvements to the README.md file

Dauboau opened the pull request:
Fixes #84

This PR proposes the following changes:
-More straightforward instruction to open the game
-Suggestion to run the command with the terminal in fullscreen mode when the minimum terminal size requirement has not been met by the user
-Instruction to use "Q" to skip the loading animation

Dauboau committed:
Adds the command to open the game to README.md

Dauboau committed:
Update README.md

Dauboau committed:
Update README.md

CarlosCraveiro commented:
Good PR!!. All of the proposed changes look very good and actually improve the clarity of the README.md.

artP2 committed:
Update R key to match it's current operation mode.

The 'r' key is current only restarting the game on the game over screen, to prevent accidental restarts.
Also the 'q' key is only quitting on the initial and gameover screens, for the same reason.
This need a discussion of how they have to operate.

artP2 reviewed (approved):
Everything is ok with the current operation of Q and R keys, but will need alteration if the community decides for a different approach on that keys.
